app/workers/metrics/record_daily_usage_worker.rb

```diff
@@ -4,14 +4,44 @@ class RecordDailyUsageWorker
     sidekiq_options queue: :low_priority, retry: 10
 
     def perform
-      articles_min_15_score_past_24h = Article.published.where("score >= 15").size
+      articles_min_15_score_past_24h = Article.published.where("score >= ? AND published_at > ?", 15, 1.day.ago).size
```

Comment:
could you add a line of comment before all of these metrics? I know they seem obvious right now but I think they won't be unless familiar with how the site operates after a while.

Especially because the description of what they are are in this and the previous PRs but not in the code

Something like

```ruby
# this metric is about this and that
articles_min_15_score_past_24h = ...

# this metric is about this and that
first_articles_past_24h = ...
```

It would be a great help!

Reply:
Good call